instrumentation/runtime-metrics/library/src/main/java/io/opentelemetry/instrumentation/runtimemetrics/Cpu.java

```diff
@@ -0,0 +1,135 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.instrumentation.runtimemetrics;
+
+import io.opentelemetry.api.OpenTelemetry;
+import io.opentelemetry.api.metrics.Meter;
+import java.lang.management.ManagementFactory;
+import java.lang.management.OperatingSystemMXBean;
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+import java.util.function.Supplier;
+import javax.annotation.Nullable;
+
+/**
+ * Registers measurements that generate metrics about CPU.
+ *
+ * <p>Example usage:
+ *
+ * <pre>{@code
+ * Cpu.registerObservers(GlobalOpenTelemetry.get());
+ * }</pre>
+ *
+ * <p>Example metrics being exported:
+ *
+ * <pre>
+ *   process.runtime.jvm.system.cpu.load_1m 2.2
+ *   process.runtime.jvm.system.cpu.utilization 0.15
+ *   process.runtime.jvm.cpu.utilization 0.1
+ * </pre>
+ */
+public final class Cpu {
+
+  // Visible for testing
+  static final Cpu INSTANCE = new Cpu();
+
+  private static final String OS_BEAN_J9 = "com.ibm.lang.management.OperatingSystemMXBean";
+  private static final String OS_BEAN_HOTSPOT = "com.sun.management.OperatingSystemMXBean";
+  private static final String METHOD_PROCESS_CPU_LOAD = "getProcessCpuLoad";
+  private static final String METHOD_CPU_LOAD = "getCpuLoad";
+  private static final String METHOD_SYSTEM_CPU_LOAD = "getSystemCpuLoad";
+
+  @Nullable private static final Supplier<Double> processCpu;
+  @Nullable private static final Supplier<Double> systemCpu;
+
+  static {
+    OperatingSystemMXBean osBean = ManagementFactory.getOperatingSystemMXBean();
+    Supplier<Double> processCpuSupplier = methodInvoker(osBean, OS_BEAN_J9, METHOD_PROCESS_CPU_LOAD);
+    if (processCpuSupplier == null) {
+      processCpuSupplier = methodInvoker(osBean, OS_BEAN_HOTSPOT, METHOD_PROCESS_CPU_LOAD);
+    }
+    processCpu = processCpuSupplier;
+
+    Supplier<Double> systemCpuSupplier = methodInvoker(osBean, OS_BEAN_J9, METHOD_SYSTEM_CPU_LOAD);
+    if (systemCpuSupplier == null) {
+      systemCpuSupplier = methodInvoker(osBean, OS_BEAN_HOTSPOT, METHOD_CPU_LOAD);
```

Comment:
If using the com.sun.management bean, [getCpuLoad](https://docs.oracle.com/en/java/javase/14/docs/api/jdk.management/com/sun/management/OperatingSystemMXBean.html#getCpuLoad()) is preferred to [getSystemCpuLoad()](https://docs.oracle.com/en/java/javase/14/docs/api/jdk.management/com/sun/management/OperatingSystemMXBean.html#getSystemCpuLoad()) as of java 14.

Reply:
I think this would be a good code comment 👍